flow/workflows/snapshot_flow.go

```diff
@@ -33,14 +33,6 @@ type SnapshotFlowExecution struct {
 	tableNameSchemaMapping map[string]*protos.TableSchema
 }
 
-type cloneTablesInput struct {
```

Comment:
why removed?

Reply:
Unnecessary

Reply:
had implemented it because getting arguments mixed around is quite possible with this call